docker-compose.dev.yml

```diff
@@ -6,9 +6,9 @@ services:
       - /bin/bash
       - -c
       - |
-        until </dev/tcp/pg/5432
+        until </dev/tcp/postgres/5432
```

Comment:
Thank you, sorry I forgot about this guy in #362!

Reply:
No problem!